TwitterTesting/Program.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This doesn't seem necessary here. Tests should be added in the UnitTests project (or a new unit test project for .net standard projects)